biz.ganttproject.core/src/biz/ganttproject/core/model/task/TaskDefaultColumn.java

```diff
@@ -50,7 +50,8 @@
   ID(new ColumnList.ColumnStub("tpd10", null, false, -1, 20), Integer.class, "tableColID", Functions.NOT_EDITABLE),
   OUTLINE_NUMBER(new ColumnList.ColumnStub("tpd11", null, false, 4, 20), String.class, "tableColOutline", Functions.NOT_EDITABLE),
   COST(new ColumnList.ColumnStub("tpd12", null, false, -1, 20), Double.class, "tableColCost"),
-  RESOURCES(new ColumnList.ColumnStub("tpd13", null, false, -1, 20), String.class, "resources", Functions.NOT_EDITABLE);
+  COST2(new ColumnList.ColumnStub("tpd13", null, false, -1, 20), Double.class, "tableColCost"),
```

Comment:
I can't test because CostAlgorithmImpl2 is missing, but here you're using the same name key `tableColCost` in your new column. That might be the reason of the troubles with the table.